src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -332,3 +333,17 @@ def streams_for_huc12s(huc12s, drb=False):
         cursor.execute(sql, [tuple(huc12s)])
 
         return [row[0] for row in cursor.fetchall()]  # List of GeoJSON strings
+
+
+def drexel_fast_zonal(geojson, dataset):
```

Comment:
It may be valuable to add a small docstring comment here to indicate that `dataset` is a dataset name. You may also consider changing to `dataset_name`, but the variable is used extensively and can probably be cleared up by a comment.